operator/docs/user-guides/forwarding_logs_to_gateway.md

```diff
@@ -134,6 +134,59 @@ _Note: While this document will only give instructions for two methods of log fo
 
     _Note:_ You can add/remove any pipeline from the ClusterLogForwarder spec in case if you want to limit the logs being sent.
 
+## Network Observability
+
+[Network Observability](https://github.com/netobserv/network-observability-operator) also require an external loki instance and is compatible with LokiStack Gateway.
+
+* Deploy the Loki Operator and `lokistack` using `openshift-network` mode and `lokistack-network` name for example. You must use a separate instance than `openshift-logging` one. 
+
+* Create the following `ClusterRole` and `ClusterRoleBinding` which allow `flowlogs-pipeline` and `network-observability-plugin` service accounts to read and write the network logs:
+
+    ```yaml
+  ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: lokistack-network-tenant-logs
+    rules:
+    - apiGroups:
+      - 'loki.grafana.com'
+      resources:
+      - network
+      resourceNames:
+      - logs
+      verbs:
+      - 'get'
+      - 'create'
+  ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: lokistack-network-tenant-logs
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: lokistack-network-tenant-logs
+    subjects:
+    - kind: ServiceAccount
+      name: flowlogs-pipeline
+      namespace: network-observability
+    - kind: ServiceAccount
+      name: network-observability-plugin
+      namespace: network-observability
+    ```
+
+* Deploy the [Network Observability Operator](https://github.com/netobserv/network-observability-operator) following the [Getting started documentation](https://github.com/netobserv/network-observability-operator/blob/main/README.md#getting-started) either from Operator Hub or commands available in the repository.
+
+* Apply the following configuration in `FlowCollector` for `network` tenant of `lokistack-network`:
+```yaml
+  loki:
+    tenantID: network
+    sendAuthToken: true
+    url: 'https://lokistack-network-gateway-http.openshift-logging.svc.cluster.local:8080/api/logs/v1/network/'
+```
+Check [config samples](https://github.com/netobserv/network-observability-operator/tree/main/config/samples) for all options.
```

Comment:
Per our live discussion I had the notion that you are not going to re-use the LokiStack instance on `openshift-logging` but rather install your LokiStack instance` on `network-observability` ns. Right?

If I am right I suggest to add steps to install the LokiStack manually on `network-observability` ns here.

Reply:
I have added the manual steps, however, the target for GA is an automatic installation and configuration of Network Observability dependent operators (loki, kafka, prometheus, etc)